In [1]:
import math

<pre>
Questions 1 & 2 refer to the following information:
A pizza chain claims that their large pizzas are at least 14 inches in diameter. You decide to test this
claim, so you buy 41 of their pizzas and measure each one. The average diameter of the pizzas you
purchased is 13.88 inches with a standard deviation of 0.40 inches.
</pre>

<pre>
1) Do you have enough evidence to refute the pizza chain’s claim with 99% certainty
</pre>

In [2]:
# Ho: mean >= 14
# Ha: mean < 14
# one tail test, 40 degrees of freedom, alpha = 0.01
# t(40, 0.01) = -2.423
t = (13.88 - 14) / (0.4/math.sqrt(41))
t

-1.9209372712298418

No.  
-1.92 > -2.423

<pre>
2) How likely is it that you would have observed these results if the pizza chain’s claim was true?
</pre>

In [ ]:
# find p value, about 3.1%

<pre>
Questions 3 & 4 refer to the following information:
At St. Garth’s hospital, when patients afflicted with a certain disease are given the standard treatment,
20% show improvement in their symptoms. A new treatment for the disease has gained popularity at
other hospitals, so the physicians at St. Garth’s decide to try the new treatment with 100 of their
patients to test whether there is any difference in the new treatment’s effectiveness.
</pre>

<pre>
3) If 27 of the 100 patients given the new treatment show improvement, is this enough evidence to
conclude with 95% certainty that the new treatment has a significantly different effect than the old?
</pre>

In [6]:
# Ho: p = 0.2
# Ha: p != 0.2
# two tail test, alpha = 0.05
z = -1.96
z_test = (.27 - .2) / math.sqrt((0.2*0.8)/100)
z_test

1.7500000000000002

No.  
abs(z_test) not greater than abs(z)

<pre>
4) If the same percentage of patients given the new treatment showed improvement using a larger
sample group, how many patients would St. Garth’s need to include in the sample in order to show
evidence of a significant difference using the new treatment with 99% confidence?
</pre>

In [7]:
# two tail test, alpha = .005
# Z(.005) = - 2.575
# -2.575 = (.27 - .2) / math.sqrt((0.2*0.8)/n)
n = 217

<pre>
Questions 5 & 6 refer to the following information:
You recently championed the purchase of a rather expensive customer relationship management (CRM)
system at your company by guaranteeing to the board of directors that the new CRM would significantly
increase the productivity of the company’s sales force. Now that the CRM system has been in place for
12 weeks, you really need to show compelling evidence that the productivity of the sales team is up.
Before the CRM came online, the sales team spoke with, on average, 3563 potential customers each
week, with a standard deviation of 326. Since the CRM came online, the sales team has recorded the
following weekly contact numbers:
3067
2998
3565
3461
3903
4190
3925
4311
4014
3908
4070
3612
</pre>

<pre>
5) Do these numbers constitute enough evidence to say with 95% confidence that the sales team has
been speaking with more potential customers per week since the implementation of the CRM?
</pre>

In [8]:
# one tail test, alpha = .05
# Ho <= 3563
# Ha > 3563
# Z(.05) = 1.645
x = 3752
mean = 3563
n = 12
z = (x - mean) / (326/math.sqrt(n))
z

2.0083288504939745

yes, outperforming  
2 > 1.645

<pre>
6) If your follow-up presentation to the board of visitors is actually still a week away (13 weeks after the
launch of the CRM), what is the minimum number of contacts that the sales team will need to report for
Week 13 to ensure that you can offer evidence that the CRM has increased productivity with 95%
confidence?
</pre>

In [10]:
# (x - 3563) / (326/sqrt(13)) = 1.645
x = 3711.73
# 3711.73 = ((3752*12) + W13) / 13
W13 = 3229

<pre>
Questions 7 & 8 refer to the following information:
In a blind taste test between two brands of root beer (Brand A and Brand B), 263 out of 500 testers
indicated that they preferred the taste of Brand B.
</pre>

<pre>
7) Does this test offer evidence, significant at 90%, that root beer drinkers prefer the taste of Brand B?
</pre>

In [ ]:
# one tail test, alpha = 0.1
# Ho: p <= 0.5
# Ha: p > 0.5
# z = 1.28
# z_measured = ((263/500) - 0.5) / sqrt((0.5*0.5)/500) = 1.16
# No evidence to reject null hypthesis, 1.16 < 1.28

<pre>
8) What is the p-value for this test?
</pre>

In [ ]:
# P(z > 1.16) = 1 - 0.877 = 0.123

<pre>
Questions 9 & 10 refer to the following information:
A factory has traditionally shipped their products to a distribution center via truck. A new railroad line
has recently been built that makes shipment between the factory and the distribution center possible
entirely by rail. The average time it takes to move products between the factory and distribution center
via truck is 406 minutes.
To move products by rail, the shipments must travel by two different rail lines, A & B, transitioning from
Line A to Line B in the middle of the journey. Thus, the total time required to travel by rail is the sum of
the times it takes to travel from the factory to the connecting point via Line A plus the time is takes to
travel from the connecting point to the distribution center via Line B. So far, 25 shipments have been
made by rail. In those 25 shipments, the average time it has taken to travel on Line A has been 213
minutes with a standard deviation of 25 minutes. The average time it has taken to travel on Line B has
been 177 minutes with a standard deviation of 17 minutes. Times for Lines A and B are independent.
</pre>

<pre>
9) Is there enough evidence to conclude with 99% certainty that shipping by rail is faster?
</pre>

In [ ]:
# one tail test
# Ho: mean >= 406
# Ha: mean < 406
# t(24, 0.01) = -2.492

# E(A+B) = 213 + 177 = 390
# Var(A+B) = 25^2 + 17^2 + 2*0*25*17 = 914 # lines are independent
# sd(A+B) = sqrt(914) = 30.232
# z_observed = (390-406) / (30.232/sqrt(25)) = -2.646
# accept Ha -> -2.646 < - 2.492

<pre>
10) If, instead of being independent, the travel times for Lines A and B are correlated at 0.73, is there
now enough evidence to conclude with 99% certainty that shipping by rail is faster?
</pre>

In [ ]:
# Var(A+B) = 25^2 + 17^2 + 2*0.73*25*17 = 1534.5
# sd(A+B) = sqrt(1534.5) = 39.173
# z_observed = (390-406) / (39.173/sqrt(25)) = -2.042
# now not enough evidence to support Ha

<pre>

</pre>